crates/cli/src/commands/manage.rs

```diff
@@ -188,6 +203,11 @@ impl Options {
                     .await?
                     .context("User not found")?;
 
+                if !ignore_complexity && !password_manager.is_password_complex_enough(&password)? {
+                    error!("That password is too weak.");
+                    return Ok(ExitCode::from(1));
```

Comment:
What's the benefit compared to just throwing here? e.g. 
```suggestion
                    anyhow::bail!("That password is too weak.");
```

This would also print the error AFAIK

Reply:
I kind of thought we would print the backtrace for anyhow errors. But that doesn't actually seem to be the case. So I guess we could do it that way if you prefer (but also, isn't it weird not to print the backtrace from anyhow?)

Reply:
It doesn't show a backtrace, it prints the Error type using `Debug`: https://doc.rust-lang.org/stable/src/std/process.rs.html#2439-2449

Which for anyhow means showing the error chain and the backtrace only if the feature is enabled (which afaik isn't the case): https://docs.rs/anyhow/latest/src/anyhow/fmt.rs.html#20-66

Anyway I'm fine with using ExitCode now